bin/dasht-docsets-inherit

```diff
@@ -0,0 +1,72 @@
+#!/bin/sh -e
+#
+# # DASHT-DOCSETS-INHERIT 1       2020-05-16                            2.4.0
+#
+# ## NAME
+#
+# dasht-docsets-inherit - unpack docsets from Dash.app
+#
+# ## SYNOPSIS
+#
+# `dasht-docsets-inherit` [*NAME*...]
+#
+# ### Examples
+#
+# `dasht-docsets-inherit`
+#   Unpack all installed Dash.app docsets.
+#
+# `dasht-docsets-inherit` sh
+#   Unpack Dash.app docsets whose names contain "sh".
+#
+# `dasht-docsets` sh 'c$'
+#   Unpack Dash.app docsets whose names contain "sh" or end in "c".
+#
+# ## DESCRIPTION
+#
+# Unpack specified Dash.app docsets. This is necessary to use the Docsets with
+# `dasht`. Docsets are specified by passing one or more name filters as
+# arguments, which are matched against docset names. If no filters are
+# specified, all docsets are unpacked. Note that `DASHT_USE_DASH_APP` must be
+# set to `true` for this script to execute.
+#
+# ## ENVIRONMENT
+#
+# `DASHT_USE_DASH_APP`
+#   Set to "true" to use Dash.app's (MacOS only) docset library. Disables
+#   docset management commands (install, remove, update). Also changes the
+#   default `DASHT_DOCSETS_DIR` to "~/Library/Application Support/Dash".
+#
+# ## EXIT STATUS
+#
+# 44
+#   No results were found.
+# 46
+#   Command was not executed because `DASHT_USE_DASH_APP` is not set.
+#  
+# ## SEE ALSO
+#
+# dasht-docsets(1), [Dash]
+#
+# [Dash]: https://kapeli.com/dash
+#
+# ## AUTHOR
+#
+# Written in 2016 by Suraj N. Kurapati <https://github.com/sunaku/dasht>
+# Distributed under the terms of the ISC license (refer to README file).
+
+: ${DASHT_USE_DASH_APP:=false}
+if $DASHT_USE_DASH_APP; then
+  : ${DASHT_DOCSETS_DIR:="$HOME/Library/Application Support/Dash"}
```

Comment:
No need for the user to set the `DASHT_USE_DASH_APP` flag; this inheritance script should simply inherit from Dash if it's found.  This would pave the way for further enhancement of this script in the future, say, to inherit from additional sources such as Zeal docsets and custom user docsets.